### PR TITLE
feat: random update - vue-mastery-card, - drawer info banner and fix issues after rebase

### PR DESCRIPTION
**Pull request by n05la3**
~~Opening this as a draft, while waiting for assets.~~

### CODE REVIEW

File: docs/src/assets/menu.json [hunk 1]
```diff
@@ -1,4 +1,27 @@
 [
+  {
+    "name": "Docs",
```
Comment (IlCallo) [inline docs/src/assets/menu.json]: This should be "Docs homepage" when shown into the drawer menu

File: docs/src/assets/menu.json [hunk 2]
```diff
@@ -1,4 +1,27 @@
...
+    "icon": "description",
+    "path": "docs"
+  },
+  {
+    "name": "Announcements",
+    "icon": "info",
+    "path": "https://github.com/quasarframework/quasar/discussions/categories/announcements",
+    "external": true
+  },
+  {
```
Comment (IlCallo) [inline docs/src/assets/menu.json:13]: Move this to the end of the array, as this isn't important

**Review by IlCallo — CHANGES_REQUESTED** [review]
Side question: why is VueMastery logo into "homepage" assets folder?

**Comment by IlCallo** [comment]
About VueMastery cards, paddings aren't the correct ones, the button text doesn't have bold weight and VueMastery icon isn't the one from mockups, maybe you need a custom one from Nick?

**Comment by n05la3** [comment]
About the vue-mastery-logo being in `homepage`, I have moved it to the root of assets. Can't find any strong reasons for having placed it in that folder.

**Comment by IlCallo** [comment]
Seems everything is ok now, perfect!

**Pull request merged**